src/csvcubeddevtools/behaviour/csvlint.py

```diff
@@ -42,7 +42,7 @@ def step_impl(context):
     temp_dir = get_context_temp_dir_path(context)
     for file in temp_dir.rglob("*.csv-metadata.json"):
         exit_code, logs = _run_csvlint(temp_dir / file)
-        assert exit_code == 0
+        assert exit_code == 0, logs
```

Comment:
Can we not assert anything specific inside the log? For example, if the csv2rdf is success, do we store a status code in the log? For example, previously we were using assert expected in logs

Reply:
As previously mentioned, these are not themselves tests. They are step definitions reused in other tests. Checking the status code is the most reliable way of ensuring an application has succeeded.